test/fixtures/mson-mixin-nonexistent.apib

```diff
@@ -9,4 +9,4 @@
 
 ## User (object)
 - id
-- Include Address
+- Include XXX
```

Comment:
All tests in `test/test-Refract.cc` calling `HandleFixtureTest()` with second parameter `false` returns error while parsing.
It is part of checking edge condition